helpers/priceconverter_helper.js

```diff
@@ -0,0 +1,15 @@
+const Helper = require('@codeceptjs/helper');
+
+class PriceConverter extends Helper {
+  parsePrice(fieldText) {
```

Comment:
custom helpers are not synchronized by framework, so you need to add `async` here, and handle returned Promise, e.g. via `await` in paces, where you use this function